### PR TITLE
Document helm executor plugin

### PR DESCRIPTION
**Pull request by mszostok**
## Description

The current documentation structure is divided into 2 parts:
- configuration
- usage

I didn't want to change the whole documentation structure, so I just added a dedicated **Executor** "sub-category" to each part.

```
- configuration
  |- executor
     |- kubectl
     |- helm
- usage
  |- executor
     |- kubectl
     |- helm
```

Later, we can rethink this approach and consider aggregating all information about a given plugin to mimic “plugin catalog” experience. For example:
```
- plugins
  |- kubectl
    |- configuration
    |- usage
  |- helm
    |- configuration
    |- usage
  |- argo
    |- configuration
    |- usage
  |- etc...
    |- configuration
    |- usage
```

Maybe also display all plugins as “tiles”. For example: https://docusaurus.io/showcase (example provided by @pkosiec 😄)

Changes proposed in this pull request:

- Adjust documentation structure.
- Document helm plugin.
- Adjust installation commands.

Preview URLs:
- https://add-helm-docs.botkube-docs.pages.dev/next/usage/executor/helm
- https://add-helm-docs.botkube-docs.pages.dev/next/configuration/executor/helm

## Related issue(s)

- https://github.com/kubeshop/botkube/issues/843

### CODE REVIEW

File: docs/usage/executor/index.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
+---
```
Comment (mszostok) [inline docs/usage/executor/index.md:1]: this is moved from https://github.com/kubeshop/botkube-docs/blob/main/docs/usage/executing-commands.md

It needs to be adjusted after merging https://github.com/kubeshop/botkube/pull/905

File: docs/usage/executor/kubectl.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
+---
```
Comment (mszostok) [inline docs/usage/executor/kubectl.md:1]: this is moved from https://github.com/kubeshop/botkube-docs/blob/main/docs/usage/executing-commands.md

File: docs/configuration/executor/helm.md [hunk 3]
```diff
@@ -0,0 +1,92 @@
+---
+id: helm
+title: Helm
+sidebar_position: 3
+---
+
+The Botkube Helm executor plugin allows you to run the `helm` command directly in the chat window of each communication platform.
+
+The Helm plugin is hosted by the official Botkube plugin repository. To enable the Helm plugin, make sure that the `botkube` repository is defined `plugins` the [values.yaml](https://github.com/kubeshop/botkube/blob/main/helm/botkube/values.yaml) file.
```
Comment (josefkarasek) [inline docs/configuration/executor/helm.md:9]:
```suggestion
The Helm plugin is hosted by the official Botkube plugin repository. To enable the Helm plugin, make sure that the `botkube` repository is defined under `plugins` in the [values.yaml](https://github.com/kubeshop/botkube/blob/main/helm/botkube/values.yaml) file.
```

File: docs/configuration/executor/index.md [hunk 4]
```diff
@@ -0,0 +1,93 @@
+---
+id: executor
+title: Executor
+sidebar_position: 1
+---
+
+The executor configuration allows you to define multiple executor configurations that can be later referred in [communication](../communication) bindings. For example, take a look on such executor definition:
```
Comment (josefkarasek) [inline docs/configuration/executor/index.md:7]:
```suggestion
The executor configuration allows you to define multiple executor configurations that can be later referred in [communication](../communication) bindings. For example, take a look at such executor definition:
```

File: docs/configuration/executor/kubectl.md [hunk 5]
```diff
@@ -1,38 +1,22 @@
...
-id: executor
-title: Executor
-sidebar_position: 4
+id: kubectl
+title: Kubectl
+sidebar_position: 2
 ---
 
-The executor configuration allows you to define multiple executor configurations that can be later referred in [communication](./communication) bindings. For example, take a look on such executor definition:
+The `kubectl` executor allows you to run the `kubectl` command directly in chat window of each communication platform.
```
Comment (josefkarasek) [inline docs/configuration/executor/kubectl.md:7]:
```suggestion
The `kubectl` executor allows you to run the `kubectl` command directly in the chat window of each communication platform.
```

File: docs/usage/executor/helm.md [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+
+```
+@Botkube helm [command] [flags]
+```
+
+At least one `helm` executor needs to be enabled and bound to a given channel.
+
+## Supported commands
+
+The Helm executor plugin has the exact same syntax as the Helm CLI. However, no all commands and flags are supported. If not supported flag is specified, you will get a dedicated error, e.g:
```
Comment (josefkarasek) [inline docs/usage/executor/helm.md:19]:
```suggestion
The Helm executor plugin has the exact same syntax as the Helm CLI. However, not all commands and flags are supported. If an unsupported flag is specified, you will get a dedicated error, e.g:
```

**Review by josefkarasek — APPROVED** [review]
lgtm

just some minor nits

**Comment by mszostok** [comment]
Thanks @josefkarasek for the review 🙇 all suggestion were applied 👍

**Pull request merged**